src/Stepper/Stepper.js

```diff
@@ -41,7 +41,6 @@ class Stepper extends Component {
 
   static defaultProps = {
     orientation: 'horizontal',
-    activeStep: 0,
```

Comment:
@oliviertassinari We should mark this with a breaking change label if we keep this in -- if you check the prop description it actually infers something different. 

The alternative is I revert this change, and we make it in a later PR. That might be a better idea to get the other changes in.